app/scripts/lib/channels/duplex.js

```diff
@@ -155,7 +155,7 @@ define([
      * @param {Object} [data]
      * @return {String}
      */
-    createMessageId: function (command, data) { //jshint ignore: line
+    createMessageId: function (command, data) {
```

Comment:
I think we already are checking for unused-variables, the rule "no-unused-vars" is set to [2, {"vars": "all", "args": "none"}], which should theoretically throw an error unless the variable was 'read' somewhere. In this file, the variables are in fact read in the following lines, where they are assigned to the obj-literal, so even with turning "args" on, we wont be getting an error i guess. Do you want me to change the "args" parameter to "all"?
@shane-tomlinson